SVS/model/train.py

```diff
@@ -163,50 +163,107 @@ def train(args):
     else:
         device = torch.device("cpu")
         logging.info("Warning: CPU is used")
+        
+    if args.vocoder_category == "pyworld":
+        train_set = SVSDataset(
+            align_root_path=args.train_align,
+            pitch_beat_root_path=args.train_pitch,
+            wav_root_path=args.train_wav,
+            pw_f0_root_path=args.train_pw_f0,
```

Comment:
I think we can uniform this right? Don't need to check vocoder category here, but check it inside the initialization of SVSDataset (__init__)